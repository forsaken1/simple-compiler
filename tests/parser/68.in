{
    int i;
    float f;
    char c;
}