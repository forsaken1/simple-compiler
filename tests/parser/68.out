{stmt}
|
+---<decl>
    |
    +---(i)
    |
    +---<decl>
        |
        +---(f)
        |
        +---(c)
